spec/cypress/integration/media_object_spec.js

```diff
@@ -15,105 +15,118 @@
  */
 
 context('Media objects', () => {
-  const media_object_id = Cypress.env('MEDIA_OBJECT_ID_2');
-  const media_object_title = Cypress.env('MEDIA_OBJECT_TITLE_2');
-  const caption = Cypress.env('MEDIA_OBJECT_CAPTION_2');
+  const media_object_id = Cypress.env('MEDIA_OBJECT_ID');
+  const media_object_title = Cypress.env('MEDIA_OBJECT_TITLE');
+  const caption = Cypress.env('MEDIA_OBJECT_CAPTION');
 
   beforeEach(() => {
     cy.login("administrator")
+    cy.intercept({
+      method: 'GET',
+      url: /\/media_objects\/.*\/section\/(?!undefined).*\/stream/
```

Comment:
We got rid of javascript calls to `/media_objects/<media_object_id>/section/<section_id>/stream` in #6240.  Maybe watching for the IIIF manifest call would be a good replacement?
```suggestion
      url: /\/media_objects\/.*\/section\/(?!undefined).*\/manifest.json/
```

Reply:
The sequence of calls that happen before the player is ready to stream and the buttons enabled should be:

1. `/media_objects/<media_object_id>`
2. `/media_objects/<media_object_id>/manifest.json`
3. `/master_files/<section_id>/auto.m3u8`
4. Request for m3u8 from streaming server
5. Request for ts chunk from streaming server

I saw a 500ms to 1s gap between #3 completing and #5 completing but it could potentially be longer.  The url pattern for 4 and 5 will be different depending on the streaming server (avalon-dev is different from mco-staging) but it will always end with `<filename>.ts?token=<streaming_token>`.

```suggestion
      url: /\.ts\?token=\w+$/
```